cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -1784,6 +1787,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["volumeattachments/status"]
+    verbs: ["patch"]
```

Comment:
What are the consequences of not updating the RBAC if the new CSI images are updated? In a patch release, updating RBAC should be optional. If the RBAC settings are necessary for the CSI driver to avoid failing, we will need to instead follow the pattern:
- The default CSI sidecars would still be the same as in v1.4
- RBAC can be optionally applied for those who want to use the new sidecars

We likely need a separate PR for the 1.4 backport to keep the same sidecars by default to avoid RBAC updates, but in master keep this PR with the new sidecars by default.

Reply:
Per discussion with @Madhu-1 the intention isn't to backport this, sounds good to make the change in master.

Reply:
As its having breaking changes and RBAC changes and as we are not planning to backport to 1.4 hope this should be fine.